src/kernel/lthread/sync/mutex.h

```diff
@@ -27,15 +27,15 @@
  *   to be waken up by mutex->wq when other schedee frees mutex.
  */
 static inline int mutex_trylock_lthread(struct lthread *self,
-					struct mutex *mutex) {
+		struct mutex *mutex) {
 	return WAITQ_WAIT_LTHREAD(self, &mutex->wq, ({
-			int done;
-			done = (mutex_trylock_schedee(&self->schedee, mutex) == 0);
-			if (!done) {
-				mutex_priority_inherit(&self->schedee, mutex);
-			}
-			done;
-		}));
+				int done;
```

Comment:
Well, technically, it's a continuation of a parameter, so we indent it with two tabs, so that's kind of logical. I'll look into whether this behaviour can be changed.